src/lib/Api/EventTracking.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzRecommendationClient\Api;
+
+use Symfony\Component\HttpFoundation\Request;
+
+class EventTracking extends AbstractApi
+{
+    const API_NAME = 'eventTracking';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getRawEndPointUri(): string
+    {
+        return 'https://event.yoochoose.net/api/%d/rendered/%s/%d/';
```

Comment:
The endpoint should be injected since it's configurable (ref: `ez_recommendation.tracking.api_endpoint`)